### PR TITLE
Youtube webpage mockup

### PR DESCRIPTION
**Pull request by AntonyOtero**
# YouTube Webpage
## Navigation bar

- It has an embedded image that represents a logo.
- The logo is aligned to the leftmost part of the navbar.
- It has an input tag that represents a search bar.
- The search bar is aligned to the middle of the navbar.
- It has a button tag that represents the search button.
- The search button is at the end (rightmost part) of the search bar.
- It has at least three embedded images on the rightmost part of the navbar that represent icons and/or an avatar.

## Video

- It is displayed.
- It shows the video title.
- It shows the number of views.
- It has like and dislike icons.
- It has a share button.
- It shows the avatar of the user who uploaded the video.
- It has a subscribe button.

## Sidebar Recommended videos

- It shows the recommended video’s thumbnail preview (as image not as an actual video).
- It shows the title of the video.
- It shows the number of views.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,8 +4,384 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
```
Comment (jubaan) [inline index.html]: - [x] Change the title to something more descriptive and meaningful [OPTIONAL]

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## :wave: :wave: Hello @AntonyOtero  and @AhmedAmin90  . Thanks for submitting you project!

Great thing this project has and has done well:
:100: Have a navbar with an embedded image, more than 3 icons to the left, and`<input>`, and a `<button>` for the search bar.
:100: has an embedded and working video, that has title, share icons, number of views, an avatar and a subscribe button
:100: has an aside column that contains thumbnails of video previews, and shows title and number of views.
:100: project has descriptive PR and README

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [x] MIT and Issues link in README are broken. Having working links increase the professionalism of your README.

- [x] The author section uses `@githubhandler` please change it to the actual github user name. This increases the professionalism of your READMe and also makes it easier to identify the authors.

### [OPTIONAL]  :yellow_circle:  

- [x] The clone would improve if a title was given in the head. See inline.

- [x] Use [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) for `<main>`

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Comment by AntonyOtero** [comment]
@jubaan We made the corrections to our project, do we submit another review request or will you continue the review from this current one?

**Comment by jubaan** [comment]
> @jubaan We made the corrections to our project, do we submit another review request or will you continue the review from this current one?

@AntonyOtero yes you have to submit the project for a re-review

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required

**Great job team. However, the issues page link on your readme file is still broken. Please fix it and send for a re-review.**

### Happy coding

**PS: You can reach me on slack (Rose Wanjohi).**

**Review by ghost — APPROVED** [review]
# Status: Approved

**Congratulations team and all the best in your next project 😃.**

**Pull request merged**